src/pages/landing/_types.ts

```diff
@@ -0,0 +1,36 @@
+import { ReactElement } from 'react'
+import type { ImageDataLike } from 'gatsby-plugin-image'
+
+export type ContentType = {
+    title: ReactElement
+    header?: ReactElement
+    second_title?: ReactElement
+    subtitle?: ReactElement
+    subtitle1?: ReactElement
+    subtitle2?: ReactElement
+    second_subtitle1?: ReactElement
+    subtitle_mobile1?: ReactElement
+    subtitle_mobile2?: ReactElement
+    imgsrc?: string
+    image_name?: string
+    image_alt?: string
+}
```

Comment:
since ContentType is used to type similar objects in various files, I suggest adding 2 more properties to this type in order to make it completely universal and use it in src/pages/landing/stocks/index.tsx as well instead of StocksType 🙂:
```suggestion
    icon?: string
    image_name_mobile?: string
}
```

Reply:
Good point, thanks. Resolved 🙇